index.js

```diff
@@ -43,7 +43,7 @@ async function terraform(args) {
   let terraformDoDestroy = core.getBooleanInput('terraform_do_destroy');
   const terraformLock = core.getBooleanInput('terraform_lock');
   const terraformParallelism = core.getInput('terraform_parallelism');
-  const terraformPlanDestroy = (terraformDoDestroy || core.getBooleanInput('terraform_plan_destroy')) ? '-destroy' : '';
+  const terraformPlanDestroy = (terraformDoDestroy || core.getBooleanInput('terraform_plan_destroy')) ? '-destroy' : [];
```

Comment:
```suggestion
  const terraformPlanDestroy = (terraformDoDestroy || core.getBooleanInput('terraform_plan_destroy')) ? ['-destroy'] : [];
```

Reply:
Hmm.. Nevermind. They're both valid

Reply:
Any specific reason for this other than cosmetics? It's being passed to `concat()`, which supports both arrays and strings as parameters.

Reply:
And Github being slow again :smile:

Reply:
Alternative approach is to update the `terraform` function:
```
  return await shell(terraformPath, args.filter(arg => (typeof arg === 'string' && arg.length > 0)), {
    cwd: core.getInput('terraform_directory'),
  });
```

Reply:
Errr, nope. That smells very much like a workaround, whereas just passing a properly typed parameter feels like an actual fix :wink: